### ISSUE TITLE
The game doesn't start

### ISSUE THREAD

**Issue by rdf453** [FTBTeam/FTB-App#1168]
### Modpack

FTB Genesis

### Modpack version

1.4.0

### Has the pack been modified

No

### Log Files

[latest.log](https://github.com/user-attachments/files/16099272/latest.log)

### Describe the bug

I just click Play button 
see progressing instance log 
suddenly cenceled progressing and did not run game
back to Genesis page

I try to move instance folder, change user folder name korean to english but noting change

Genesis log files were not created, so I don't know what is wrong
Plz let me know what is wrong

### Steps to reproduce

1.Go to FTB Genesis page
2.Click Play button
3.Progressed until Veioldate asset
4.Cenceled instance progressing and return to FTB Genesis page

### Expected behaviour

I Expect the game will run

### Screenshots

![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/174717873/eab0e7dc-0ff8-4378-8c69-a60442fe1752)

![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/174717873/476dcc18-0df0-46dc-bc24-82540d99e691)

### Additional information

FTB-DBGETAKAXIVOT

all mod is disabled by unknown error

Plz let me know why mod is disabled too

**Comment by Gaz492** [comment]
Do you have any special characters in your windows username?

In parts of the log it shows up correctly, while in others it shows up as `¹®¿¹ÀÎ`

**Comment by rdf453** [comment]
@Gaz492 nope user name is just my name

**Comment by Gaz492** [comment]
> @Gaz492 nope user name is just my name

Could you go to Settings > app and click the upload logs button then reply with the url?
This might give us better logs than the debug tool does (we need to release an update to it)

**Comment by rdf453** [comment]
> > @Gaz492 nope user name is just my name
> 
> Could you go to Settings > app and click the upload logs button then reply with the url? This might give us better logs than the debug tool does (we need to release an update to it)

https://pste.ch/uderaborok

here

**Comment by Gaz492** [comment]
Thanks, this does look to be an issue with reading/getting the username of your computer, we are working on a fix for this, but not sure when it will be released at the moment
!skip

**Comment by rdf453** [comment]
> Thanks, this does look to be an issue with reading/getting the username of your computer, we are working on a fix for this, but not sure when it will be released at the moment !skip

I hope it gets resolved :D

**Comment by Gaz492** [comment]
We have just released an app update that should fix these issues, if you still run into the issue, do let us know!

**Comment by rdf453** [comment]
> We have just released an app update that should fix these issues, if you still run into the issue, do let us know!

First of all, this problem was not resolved in the ftb app(overwolf) and appeared repeatedly. However, I don't know why, but I downloaded the ftb electron app (stand alone) thinking I couldn't find the update, and it ran and the update went well. The problem even disappeared with the ftb app that used to run through overwolf, which was causing problems after running and downloading it as an electron app (stand alone). I also think that maybe the overwolf could have been the problem. Thank you for your help over the long period of time and thank you for your hard work. It seems like it was resolved haphazardly, so it's a bit weird, but since all roads lead to Rome, all bugs need to be resolved, so a good thing is a good thing.
Anyway, thank you again for your help.

It may sound strange since I'm using a translator now, but the problem has definitely been solved :D
